neet/boolean/data/s_pombe-edges.txt

```diff
@@ -0,0 +1,25 @@
+SK Ste9 -1
```

Comment:
Add instruction for creating node/edge file to documentation: no space for node names